producer.py

```diff
@@ -181,6 +185,15 @@ def package_releases_update(
                     package_index.url,
                     only_if_package_seen=only_if_package_seen,
                 )
+                async_tasks.append(
+                    package_released.publish_to_topic(
+                        package_released.MessageContents(
+                            package_name=package_name,
+                            package_version=package_version,
+                            index_url=[package_index.url],
```

Comment:
please add `service_version` and `component_name` to `message_contents`

Reply:
thanks, https://github.com/thoth-station/package-releases-job/pull/449/commits/6510ac329510c4615282a350c2ef3e6855959c0c